FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,44 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImageMapper.map(data: data, response: response))
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+class FeedImageMapper {
+	static let OK_200 = 200
```

Comment:
The `OK_200` property should have a private access level since it is used only in the `FeedImageMappper`.

```suggestion
	private static let OK_200 = 200
```

Reply:
Thanks, @mapostolakis for the feedback, I've made changes as per your suggestion.

Reply:
Thanks @mapostolakis for the feedback. I've pushed a new commit as per your suggestion.